conductor-client/src/main/kotlin/com/openlattice/graph/partioning/RepartitioningJob.kt

```diff
@@ -92,7 +92,7 @@ class RepartitioningJob(
 
     @JsonIgnore
     override fun setDataSourceResolver(resolver: DataSourceResolver) {
-        this.hds = resolver.getDefaultDataSource()
+        this.hds = resolver.resolve(state.entitySetId)
```

Comment:
Just making sure I'm on the same page ... once we move everything to aurora, the repartition stuff will no longer need to exist, right?

Reply:
That is correct. We can just remove it all.